### ISSUE TITLE
Unable to import in v13.9.0+ due to default export condition order

### ISSUE THREAD

**Issue by twgraham** [spencermountain/compromise#815]
Every since version 13.9.0, my project refuses to build in webpack when I try import `compromise`. I get the following ambiguous message:
`Module not found: Error: Default condition should be last one`

The stack trace was pretty useless, so I needed to track down the issue manually. It seems that webpack does not like the `default` export condition order.
https://github.com/spencermountain/compromise/blob/bd7929c64cf9cb0dbd7fd38c273d9bef7a5feafc/package.json#L12-L21

According to webpack, the default case [should be last](https://webpack.js.org/guides/package-exports/#conditional-syntax)... beats me 🤷‍♂️ 

If i change the export group so that default is last like so, webpack is happy...

```json
"exports": {
    ".": {
      "require": "./builds/compromise.js",
      "import": "./builds/compromise.mjs",
      "default": "./builds/compromise.js"
    }
}
```

**Comment by spencermountain** [comment]
oh my gosh, thank you.
will fix this today.

**Comment by spencermountain** [comment]
released as `13.9.3` - thank you for the heads-up!
if you see anything else, pls let me know. Webpack is a journey.
